twill-core/src/main/java/org/apache/twill/internal/TwillRuntimeSpecification.java

```diff
@@ -40,11 +41,13 @@
   private final int reservedMemory;
   private final String rmSchedulerAddr;
   private final LogEntry.Level logLevel;
+  private final Map<String, Map<String, LogEntry.Level>> logLevels;
 
   public TwillRuntimeSpecification(TwillSpecification twillSpecification, String fsUser, URI twillAppDir,
                                    String zkConnectStr, RunId twillRunId, String twillAppName,
                                    int reservedMemory, @Nullable String rmSchedulerAddr,
-                                   @Nullable LogEntry.Level logLevel) {
+                                   @Nullable LogEntry.Level logLevel,
```

Comment:
better rename this as `rootLogLevel`, just like in other classes. Also rename the getter.